### DISCUSSION TITLE
TTS Model creation execution time in stage 7

### DISCUSSION THREAD

**Discussion by murali5999** [espnet/espnet#5888] [General]
Hello team, request you to let me know how long does model execution time takes when I am using the following hard ware with one  GPU card. If I type command nvidi-smi GPU is fully processing not giving details.
Unable to determine the device handle for GPU0000:01:00.0: Unknown Error

Thanks..
![espnet-execution time](https://github.com/user-attachments/assets/8b438503-3a64-4940-ba03-42bf35a5b7c2)
![GPU card details](https://github.com/user-attachments/assets/3e9f0ab8-d01e-4fb8-b468-4bb63b1b518c)

**Comment by sw005320** [comment]
It depends on the model size and amount of data. 
So, please describe them in detail

**Reply by murali5999** [reply]
Thank you Shinji Watanabe for your response. I am using default LJspeech data set given in the espnet execution stage 0 with command ./run.sh from the folder /espnet/egs2/ljspeech/tts1. downloaded deafault speech dataset of wav files and then processed all the batch.
For this how long the execution time takes to create model.pth files.
In case if we want to reduce execution time with limited set of train wav files. then how can we do customize the script? Please explain. the training it self taking days time, how can we check GPU is executing properly. is there any way to execute model in 2 to 3 hours time with limited set of train dataset.
Thanks,
Murali
